src/mono/browser/runtime/run.ts

```diff
@@ -1,14 +1,10 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-<<<<<<< HEAD
 import NativeAOT from "consts:nativeAOT";
```

Comment:
```suggestion
import NativeAOT from "consts:nativeAOT";

```
You can also separate out `Module` (only needed for NAOT) from ` import { ENVIRONMENT_IS_NODE, Module, loaderHelpers, mono_assert, runtimeHelpers } from "./globals";` with a separate `import`. Note you'll need `/* eslint-disable prefer-rest-params */`.

Reply:
Done.